<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="lidar_tp" default="/livox/lidar"/>
    <arg name="output_pcd_filename" default="cloud"/>
    <arg name="threshold_lidar" default="5"/>
    

    <node pkg="manual_calibration" name="pcdTransfer" type="pcdTransfer" output="screen">
        <param name="lidar_tp" value="$(arg lidar_tp)" /> 
        <param name="output_pcd_path" value="$(find manual_calibration)/data/pcdFiles/$(arg output_pcd_filename)" />  <!-- path to save new pcd files -->
        
        <param name="threshold_lidar" type="int" value="$(arg threshold_lidar)" />  <!-- the limit of messages to transfer to the pcd file, 80 means maximum 80 messages of lidar -->

        <param name="pcdTransferFinished" value="false"/>
    </node>

</launch>

